cmake/OmimPlatform.cmake

```diff
@@ -0,0 +1,34 @@
+set(PLATFORM_LINUX FALSE)
+set(PLATFORM_MAC FALSE)
+set(PLATFORM_WIN FALSE)
+set(PLATFORM_ANDROID FALSE)
+set(PLATFORM_IPHONE FALSE)
+set(PLATFORM_DESKTOP FALSE)
+
+if("${CMAKE_SYSTEM_NAME}" STREQUAL "Linux")
+  set(PLATFORM_LINUX TRUE)
+endif()
+
+if("${CMAKE_SYSTEM_NAME}" STREQUAL "Darwin")
+  set(PLATFORM_MAC TRUE)
+endif()
+
+if("${CMAKE_SYSTEM_NAME}" STREQUAL "Windows")
+  set(PLATFORM_WIN TRUE)
+endif()
+
+if("${CMAKE_SYSTEM_NAME}" STREQUAL "Android")
+  set(PLATFORM_ANDROID TRUE)
+endif()
+
+if("${CMAKE_SYSTEM_NAME}" STREQUAL "iOS")
+  set(PLATFORM_IPHONE TRUE)
+endif()
+
+if (PLATFORM_LINUX OR PLATFORM_MAC OR PLATFORM_WIN)
+  set(PLATFORM_DESKTOP TRUE)
```

Comment:
Setting it in the ifs above would be cleaner/simpler when adding a new platform.